sunpy/map/tests/test_maputils.py

```diff
@@ -272,3 +273,32 @@ def test_contains_coord(aia171_test_map):
     multi_coord = SkyCoord([0, 2000]*u.arcsec, [0, 2000]*u.arcsec,
                            frame=smap.coordinate_frame)
     assert (contains_coordinate(smap, multi_coord) == [True, False]).all()
+
+
+def test_extract_along_coord(aia171_test_map):
```

Comment:
This test probably needs fixing. Because the slice is along the diagonal, it does not ensure that I've not mixed up the ordering of the indices when extracting the intensity from the array.